modules/management-console-agent/src/test/java/org/apache/ignite/agent/processor/export/MetricsExporterTest.java

```diff
@@ -72,6 +72,61 @@ public void testResponseWithoutUserTag() {
         doTestResponse(userTag);
     }
 
+    /** */
+    @Test
+    public void testDeserializeResponseFromByteArray() {
+        UUID clusterId = UUID.randomUUID();
+
+        String consistentId = "testConsistentId";
+
+        Map<String, MetricRegistry> metrics = generateMetrics();
+
+        MetricResponse msg = exporter.metricMessage(clusterId, "tag", consistentId, metrics);
+
+        MetricResponse newRes = new MetricResponse(msg.body());
+
+        Map<String, Byte> map = new HashMap<>();
+
+        for (MetricRegistry reg : generateMetrics().values()) {
+            for (Metric m : reg) {
+                byte type;
+
+                if (m instanceof BooleanMetric)
+                    type = 0;
+                else if (m instanceof IntMetric)
+                    type = 1;
+                else if (m instanceof LongMetric)
+                    type = 2;
+                else if (m instanceof DoubleMetric)
+                    type = 3;
+                else
+                    throw new IllegalArgumentException("Unknown metric type.");
```

Comment:
Is there a chance that new metric types will be introduced in future?
If yes, the test will be failing - is it better than storing types in a kind of enum?

Just asking :)